src/stock-operations/add-stock-operation/add-stock-operation.component.tsx

```diff
@@ -200,7 +201,7 @@ const AddStockOperation: React.FC<AddStockOperationProps> = (props) => {
         >
           <div style={{ margin: '10px' }}>
             {isEditing && (
-              <div style={{ display: 'flex', flexDirection: 'row' }}>
+              <div style={{ display: 'flex', flexDirection: 'row', marginLeft: '10px' }}>
```

Comment:
@jabahum what do you think should we not manage css in it's own file for ease of maintainability?